src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package duke.tasks;
+
+public class Deadline extends Task {
+
+    private String type = "[D]";
+    private String dueDate;
+
+    public Deadline(String description, String dueDate) {
+        super(description);
+        this.dueDate = dueDate;
+    }
+
+    public Deadline(String description, boolean isDone, String dueDate) {
+        super(description, isDone);
+        this.dueDate = dueDate;
+    }
+
+    public String getType(){
```

Comment:
Coding standard violation: consider using Egyptian style brackets (should have space bar before curly bracket).